tasks/eva_3090/archive_to_lts.py

```diff
@@ -0,0 +1,98 @@
+import gzip
+import shutil
+import tarfile
+import os.path
+from argparse import ArgumentParser
+
+from ebi_eva_common_pyutils.logger import logging_config
+from retry import retry
+
+logging_config.add_stdout_handler()
+logger = logging_config.get_logger(__name__)
+
+
+def make_tarfile(output_filename, source_dir):
```

Comment:
Suggest swapping source and destination argument positions to be consistent with the other functions...

Reply:
Addressed that in (https://github.com/EBIvariation/eva-common-pyutils/pull/33)